packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -787,6 +787,7 @@ class _InkResponseState extends State<_InkResponseStateWidget>
 
   @override
   void dispose() {
+    _dispose();
```

Comment:
would it make sense to just inline _dispose here? I only see one call to _dispose